src/environments/__tests__/JSDOMEnvironment.js

```diff
@@ -0,0 +1,21 @@
+'use strict';
+
+jest.autoMockOff();
+const utils = require('../../lib/utils');
+const JSDOMEnvironment = require('../JSDOMEnvironment');
+const CONFIG = utils.normalizeConfig({
```

Comment:
I mean you don't need normalizeConfig at all, just pass an empty object to the new call below.